dwave/cloud/testing/mocks.py

```diff
@@ -135,6 +136,78 @@ def qpu_clique_solver_data(size: int, **kwargs) -> dict:
     return structured_solver_data(**params)
 
 
+def qpu_chimera_solver_data(m: int,
+                            n: Optional[int] = None,
+                            t: Optional[int] = None,
+                            **kwargs) -> dict:
+    """Mock QPU solver data with a custom-sized Chimera topology.
+
+    Args:
+        m:
+            Number of rows in the Chimera lattice.
+        n:
+            Number of columns in the Chimera lattice.
+        t:
+            Size of the shore within each Chimera tile.
+        **kwargs:
+            Solver properties passed down to :meth:`.structured_solver_data`.
+    """
+    try:
+        import dwave_networkx as dnx
+    except ImportError:     # pragma: no cover
+        raise RuntimeError("Can't generate Chimera graph without dwave-networkx. "
+                           "Install with 'dwave-cloud-client[mocks]'.")
+
+    graph = dnx.chimera_graph(m, n, t)
+    qubits = list(graph.nodes)
+    couplers = list(graph.edges)
+    num_qubits = len(qubits)
+
+    params = dict(
+        id=f"dw_{num_qubits}q_mock",
+        description=f"A {num_qubits}-qubit mock QPU solver with chimera topology",
+        qubits=qubits,
+        couplers=couplers,
+        topology={"type": "chimera", "shape": [m, n, t]},
+    )
+    params.update(**kwargs)
+
+    return structured_solver_data(**params)
+
+
+def qpu_pegasus_solver_data(m: int,
+                            **kwargs) -> dict:
+    """Mock QPU solver data with a custom-sized Pegasus topology.
+
+    Args:
+        m:
+            Size parameter for the Pegasus lattice.
+        **kwargs:
+            Solver properties passed down to :meth:`.structured_solver_data`.
+    """
+    try:
+        import dwave_networkx as dnx
+    except ImportError:     # pragma: no cover
+        raise RuntimeError("Can't generate Pegasus graph without dwave-networkx. "
+                           "Install with 'dwave-cloud-client[mocks]'.")
+
+    graph = dnx.pegasus_graph(m, fabric_only=False)
```

Comment:
Why `fabric_only=False` here? Online Advantage QPUs usually have qubits from 30 and up

Reply:
Good catch. I had it like this initially to match the `num_qubits`, but it makes sense to be closer to an actual online qpu.